udp/client/clientconn.go

```diff
@@ -469,6 +470,7 @@ func (cc *ClientConn) Sequence() uint64 {
 }
 
 func (cc *ClientConn) Process(datagram []byte) error {
+	cc.lastActivity = time.Now()
```

Comment:
Pls use atomic.Value to store lastActivity and move it to inactivityMonitor struct.